custom/covid/management/commands/update_cases.py

```diff
@@ -13,58 +18,66 @@ class CaseUpdateCommand(BaseCommand):
     """
 
     def __init__(self):
-        self.output_file = None
         self.extra_options = {}
 
-    def case_block(self):
+    def logger_name(self):
```

Comment:
If you make this a `@property`, you can still override it with a simple `logger_name = __name__` in the subclasses:
```python
In [1]: class Foo:
   ...:     @property
   ...:     def name(self):
   ...:         raise NotImplementedError()

In [2]: class Bar(Foo):
   ...:     name = "hello"

In [3]: Bar().name
Out[3]: 'hello'
```

Reply:
nice, I didn't know that - 6982c65ad1ea22ffb6534ef66ab977289698d360